.changeset/fix-reference-attribute-filtering.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/dashboard": patch
+---
+
+Fix reference attribute filtering in products list page after GraphQL schema changes. Product filters using reference attributes (pages, products, variants) now work correctly with latest Saleor schema.
```

Comment:
this is unreleased so we don't need a changeset. I mean this broken build never went to production so this entry is a noise